some typo fixes, more fluid readability and fixes of weird machine translated terms for the english parts of the guide (#10231)
MaaAssistantArknights/MaaAssistantArknights: docs/en-us/protocol/sss-schema.md

# Security Presence Schema

> This document is machine-translated. If you have the ability, please take a look at the Chinese version. We would greatly appreciate any errors or suggestions for improvement.

Reviewer: ```suggestion
This document is machine-translated. If you have the ability, please refer to the Chinese version. We would greatly appreciate any errors or suggestions for improvement.
```
Referring to something is much more professional, which is needed in the docs.
